docs/data-sources/vpcs.md

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "Virtual Private Cloud (VPC)"
+---
+
+# huaweicloud_vpcs
+
+Use this data source to get a list of VPC.
+
+## Example Usage
+
+An example filter by name and tag
+
+```hcl
+variable "vpc_name" {}
+
+data "huaweicloud_vpcs" "vpc" {
+  name = var.vpc_name
+  tags {
+    foo = "bar,value"
+  }
+}
+
+output "vpc_ids" {
+  value = data.huaweicloud_vpcs.vpc.vpcs[*].id
+}
+```
+
+## Argument Reference
+
+The arguments of this data source act as filters for querying the available VPCs in the current region.
+ All VPCs that meet the filter criteria will be exported as attributes.
+
+* `region` - (Optional, String) Specifies the region in which to obtain the VPC. If omitted, the provider-level region
+  will be used.
+
+* `name` - (Optional, String) Specifies the name of the desired VPC. The value is a string of no more than 64 characters
+  and can contain digits, letters, underscores (_) and hyphens (-).
+
+* `id` - (Optional, String) Specifies the id of the desired VPC.
```

Comment:
Generally, for reading habits, the `id` is in the first position of the optional arguments, and the `region` is in the first position